workflows/python/sdk/order-processor/workflow_activities.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import timedelta
+import logging
+import json
+from dapr.ext.workflow import DaprWorkflowContext, WorkflowActivityContext
+# , when_any
+from model import InventoryItem, Notification, InventoryRequest, OrderPayload, OrderResult, PaymentRequest, InventoryResult
+from dapr.clients import DaprClient
+from util import get_address
+
+store_name = "statestore-actors"
+
+address = get_address()
+
+def order_processing_workflow(ctx: DaprWorkflowContext, orderPayload: OrderPayload):
```

Comment:
We should consider restructuring the files/folders down this directory to mimic the structure inside the .NET repo. 

Inside .NET we breakout the workflow from the activities and each activity gets their own seperate file. It seems wrong to me that we have a file named 'workflow_activities.py' that also contains the definition of the workflow itself.

Reply:
Actually, did it intentionally.
In python, it's simply a `def` that is required to be defined for workflow/activities, very different from .NET, where-in different classes with different purposes are added - each catering to a separate small part of functionality.
Infact, various things are different - there is no dependency injection happening here -so, just wanted to keep it in the same file - highlighting their simplicity.

What I can do better is to have proper doc for each workflow/activity - that should help in highlighting each of them separately.

Reply:
I agree we should not copy the .NET convention. What we're doing now feels simpler and more natural for a Python project.